extensions/amp-install-serviceworker/amp-install-serviceworker.md

```diff
@@ -37,7 +37,7 @@ limitations under the License.
 
 ## Behavior
```

Comment:
I think it might be useful to explicitly say that either the `src` URL or the `data-iframe-src` URL is used, but never both. (If I'm understanding the code correctly.)

> If `src=URL1` and `data-iframe-src=URL2`, then:
> * **AMP page *not* served by cache** – when the page is *not* served by the cache, then `URL1` is registered as a service worker in the standard way via `navigator.serviceWorker.register(URL1)`.
> * **AMP page served by cache** – an iframe is created with `URL2` as the source. (This page will itself presumably register `URL1` as a service worker for the domain.)